streampark-common/src/main/java/org/apache/streampark/common/utils/UUIDUtils.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.common.utils;
+
+import lombok.experimental.UtilityClass;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+@UtilityClass
+public class UUIDUtils {
+
+    public static long generateUUID() {
+        String uuidString = UUID.randomUUID().toString().replaceAll("[^0-9]]", "");
```

Comment:
Good point. Will fix it.

Reply:
Since the maximum length is too long, I reduced the length to 14 bits. I also added multithreaded unit tests to ensure there are no conflicts. PTAL @HxpSerein

Reply:
ok